### PR TITLE
#1197 added UrlUpdater for java mission control

### PR DESCRIPTION
**Pull request by moritzLanger**
Both the official website and GitHub repo did not work with the updater, because at least one version would be missing. Sadly there wasn't a Json I could find but the Mirrored Repo of adoptium delivers every version for the GithubUrlUpdater.

### CODE REVIEW

File: url-updater/src/main/java/com/devonfw/tools/ide/url/updater/jmc/JmcUrlUpdater.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  @Override
+  protected String getGithubOrganization() {
+
+    return "adoptium";
+  }
+
+  @Override
+  protected String getGithubRepository() {
+
+    return "jmc-build";
```
Comment (hohwille) [inline url-updater/src/main/java/com/devonfw/tools/ide/url/updater/jmc/JmcUrlUpdater.java:26]: latest tag is  9.0.0-SNAPSHOT but there is no such release. Can you filter out SNAPSHOT versions?

**Review by hohwille — CHANGES_REQUESTED** [review]
@moritzLanger thanks for your PR. Looks great.
Can you check my comment regarding SNAPSHOT versions. After all the cleanups, where I had to remove content from ide-urls manually, I want to get it right from the start...

File: url-updater/src/main/java/com/devonfw/tools/ide/url/updater/jmc/JmcUrlUpdater.java (hunk 1 above)
Comment (moritzLanger) [reply url-updater/src/main/java/com/devonfw/tools/ide/url/updater/jmc/JmcUrlUpdater.java:26]: While testing locally the SNAPSHOT version already got filtered, or do you mean something else? 
image

Comment (hohwille) [reply url-updater/src/main/java/com/devonfw/tools/ide/url/updater/jmc/JmcUrlUpdater.java:26]: Thanks for checking. Resolved.

**Review by hohwille — APPROVED** [review]
My concern was wrong. All great 👍 
Ready for merge.

**Pull request merged**